README.md

```diff
@@ -1,15 +1,28 @@
-# go2xunit
+# go2xunit [![Build Status](https://travis-ci.org/tischda/go2xunit.svg)](https://travis-ci.org/tischda/go2xunit)
 
 Converts `go test -v` (or `gocheck -vv`) output to xunit compatible XML output
 (used in [Jenkins][jenkins]/[Hudson][hudson]).
 
+This is a fork from go2xunit by Miki Tebeka <miki.tebeka@gmail.com>.
```

Comment:
If you can it'll be great.

I *do* plan to do some refactoring, but that's not related.